static/templates/settings/profile_field_settings_admin.hbs

```diff
@@ -1,6 +1,8 @@
 <div id="profile-field-settings" class="settings-section" data-name="profile-field-settings">
-    <h3 class="inline-block">{{t "Custom profile fields" }}</h3>
     <div class="alert-notification" id="admin-profile-field-status"></div>
+    <div class="settings_panel_list_header">
+        <div class="list_title">{{t "Custom profile fields"}}</div>
```

Comment:
Should we call this class `settings_panel_list_title` just for consistency with the `settings_panel_list_header` class?

Reply:
I value brevity here.  I gave the container a long name to ensure it always stays unique.  The `list_title` class is also unique, and it's consistent with the short class for the search widget ("search") being short.  I understand that `list_title` may not always remain unique, but it's nested in the CSS selectors.